### ISSUE TITLE
Local client returns numpy.float64 for score (and other)

### ISSUE THREAD

**Issue by SlowMo24** [qdrant/qdrant-client#295]
When running an in-memory server, qdrant retruns numpy.float64 but for server version it returns normal floats.

```python
from typing import List

from qdrant_client import QdrantClient
from qdrant_client.grpc import ScoredPoint
from qdrant_client.http.models import Distance, VectorParams
from qdrant_client.http.models import PointStruct

def insert_n_get(client: QdrantClient) -> List[ScoredPoint]:
    client.recreate_collection(
        collection_name="test_collection",
        vectors_config=VectorParams(size=4, distance=Distance.DOT),
    )
    client.upsert(
        collection_name="test_collection",
        wait=True,
        points=[
            PointStruct(id=1, vector=[0.05, 0.61, 0.76, 0.74], payload={"city": "Berlin"}),
            PointStruct(id=2, vector=[0.19, 0.81, 0.75, 0.11], payload={"city": "London"}),
            PointStruct(id=3, vector=[0.36, 0.55, 0.47, 0.94], payload={"city": "Moscow"}),
            PointStruct(id=4, vector=[0.18, 0.01, 0.85, 0.80], payload={"city": "New York"}),
            PointStruct(id=5, vector=[0.24, 0.18, 0.22, 0.44], payload={"city": "Beijing"}),
            PointStruct(id=6, vector=[0.35, 0.08, 0.11, 0.44], payload={"city": "Mumbai"}),
        ]
    )
    return client.search(
        collection_name="test_collection",
        query_vector=[0.2, 0.1, 0.9, 0.7],
        limit=3
    )


client_deploy = QdrantClient("localhost", port=6333)
client_memory = QdrantClient(':memory:')

result_deploy = insert_n_get(client_deploy)[0].score
result_local = insert_n_get(client_memory)[0].score

print(result_deploy)
print(result_local)

print(type(result_deploy))
print(type(result_local))

assert result_deploy == result_local
```

returns 

```shell
1.362
1.362000031210482
<class 'float'>
<class 'numpy.float64'>

Traceback (most recent call last):
  File "~/test.py", line 44, in <module>
    assert result_deploy==result_local
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError
```

possibly related: #192

**Comment by joein** [comment]
it is probably more related to https://github.com/qdrant/qdrant-client/issues/163
and not related to #192

thanks for highlighting it
will look into it

**Comment by joein** [comment]
fixed in 1.6.1
